source/comp/markv_model.h

```diff
@@ -32,7 +32,46 @@ class MarkvModel {
  public:
   MarkvModel()
       : operand_chunk_lengths_(
-            static_cast<size_t>(SPV_OPERAND_TYPE_NUM_OPERAND_TYPES), 0) {}
+            static_cast<size_t>(SPV_OPERAND_TYPE_NUM_OPERAND_TYPES), 0) {
+    // Set default values.
+    operand_chunk_lengths_[SPV_OPERAND_TYPE_TYPE_ID] = 4;
```

Comment:
Normally you should briefly say how you tuned these numbers.
But perhaps it's no more than "stared at examples".
(And I see I've reviewed this bit before.)